### PR TITLE
DCOS-14851: Remove UCR option from service create form

### PR DESCRIPTION
**Pull request by jfurrow**
This PR removes the UCR option in the service create form. Check out the discussions in these tickets for more context: https://jira.mesosphere.com/browse/DCOS-14851 and https://jira.mesosphere.com/browse/DCOS-15539

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by weblancaster** [comment]
retest this please

File: plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ class ContainerServiceFormAdvancedSection extends Component {
     if (gpusDisabled) {
       inputNode = (
         <Tooltip
-          content="Docker Engine does not support GPU resources, please select Universal Container Runtime if you want to use GPU resources."
+          content="Docker Engine does not support GPU resources, please select Mesos Runtime if you want to use GPU resources."
```
Comment (weblancaster) [inline plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js]: Are we able to use i18n here?

**Comment by weblancaster** [comment]
@Poltergeist Since @jfurrow is out on vacation let's make the changes ourselves if that's fine with you.

**Comment by Poltergeist** [comment]
Yes let's do this.

**Comment by weblancaster** [comment]
retest this please. (the tests never finished 😕 )

File: plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js (hunk 1 above)
Comment (jfurrow) [inline plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js]: Sorry for not addressing this! I opted not to for this PR to keep it as simple as possible, but I agree this should be internationalized.

**Pull request merged**